riscv/src/asm/mod.rs

```diff
@@ -1,3 +1,8 @@
+// this is required to please clippy which finds dome dead code in the
```

Comment:
But the generated parser already has `allow(all)` (in parser.rs) - where exactly is the dead code?

Reply:
The parser has `allow(clippy::all)` but `dead_code` is a rustc lint not a clippy lint. I moved it there!